### DISCUSSION TITLE
xunlei云盘无法在线播放

### DISCUSSION THREAD

**Discussion by 54nzg** [AlistGo/alist#1327] [General]
![image](https://user-images.githubusercontent.com/82253076/178444102-be52fb62-e3d3-47c5-8e16-c9f735c99ac9.png)
![image](https://user-images.githubusercontent.com/82253076/178444117-e8012210-7ca4-4dfc-944b-310b5a166c52.png)

**Comment by rakaki** [comment]
这个问题貌似issue里有提到。我也这样，目前只能用webdav播放

**Comment by anwen-anyi** [comment]
首先你要了解 这个代理是做什么滴~~~~~ 其次你开启"代理"后,是由你本机(搭建Alist的机器中转)流量进行播放的 你本机的宽带不行播放自然不行,你要搞个下行速度快的服务器 或者 自己家的宽带300M的就行(我就是300M家庭宽带) 你看看能不能放.
